examples/api/amp-inputmask.js

```diff
@@ -22,43 +22,17 @@ const upload = multer();
 // eslint-disable-next-line new-cap
 const examples = express.Router();
 
-examples.all('/default', upload.none(), (request, response) => {
-  if (request.method !== 'POST') {
-    response.send('post only');
-    return;
-  }
-  const code = request.body ? request.body.code : '';
-  const codeUnmasked = request.body ? request.body.codeUnmasked : '';
-  response.json({
-    code,
-    'code-unmasked': codeUnmasked,
-  });
-});
+examples.post('/default', upload.none(), handleRequest);
+examples.post('/postal', upload.none(), handleRequest);
+examples.post('/phone', upload.none(), handleRequest);
```

Comment:
simpler: 

```
examples.post(['/default', '/postal', '/phone'] , upload.none(), handleRequest);
```

Reply:
You're right, I just changed it to your suggestion.